addons/sale/views/sale_portal_templates.xml

```diff
@@ -131,71 +131,50 @@
-                                        </a>
-                                        <a role="button" t-elif="to_accept_and_pay" class="btn btn-success btn-block" data-toggle="modal" data-target="#modalaccept" href="#">
-                                            <i class="fa fa-check"/> <t t-if="not sale_order.signature">Accept &amp; </t>Pay
-                                        </a>
-                                        <t t-if="sale_order.is_expired">
-                                            <div class="alert alert-warning small mt-3 mb-1 py-1 px-2"><b class="d-block mb-1">This offer expired!</b>
-                                                <a role="button" href="#discussion" class="btn btn-info btn-block py-1 o_normalwhitespace"> Get a new quotation.</a>
-                                            </div>
-                                        </t>
-                                    </t>
-                                </t>
-
-                                <t t-if="need_payment or (sale_order.state in ('sale') and sale_order.has_to_be_paid() and tx_state != 'done')">
-                                    <a role="button" class="btn btn-success btn-block" data-toggle="modal" data-target="#modalaccept" href="#">
-                                        <i class="fa fa-check"></i> Pay Now
-                                    </a>
-                                </t>
+                                <a t-if="sale_order.has_to_be_signed(True)" role="button" t-attf-class="btn btn-{{ 'primary' if sale_order.has_to_be_paid() else 'success'}} btn-block mb8" data-toggle="modal" data-target="#modalaccept" href="#">
+                                    <i class="fa fa-check"/> Accept &amp; Sign
+                                </a>
+                                <a t-elif="sale_order.has_to_be_paid(True)" role="button" class="btn btn-success btn-block mb8" data-toggle="modal" data-target="#modalaccept" href="#">
+                                    <i class="fa fa-check"/> <t t-if="not sale_order.signature">Accept &amp; Pay</t><t t-else="">Pay now</t>
+                                </a>
+                                <div class="o_download_pdf btn-toolbar flex-sm-nowrap">
+                                    <div class="btn-group flex-grow-1 mr-1 mb-1">
+                                        <a class="btn btn-secondary btn-block o_download_btn" t-att-href="sale_order.get_portal_url(report_type='pdf', download=True)" title="Download"><i class="fa fa-download"/> Download</a>
+                                    </div>
+                                    <div class="btn-group flex-grow-1 mb-1">
+                                        <a class="btn btn-secondary btn-block o_print_btn o_portal_invoice_print" t-att-href="sale_order.get_portal_url(report_type='pdf')" id="print_invoice_report" title="Print" target="_blank"><i class="fa fa-print"/> Print</a>
```

Comment:
Hide print in mobile like it was before the recent portal changes?
https://drive.google.com/open?id=1GHjRxL5V7-ZbI1K235kTZ7Wv4wAfLuNc
from chatter on task https://www.odoo.com/web#id=39876&action=327&model=project.task&view_type=form&menu_id=4720